scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -678,6 +678,44 @@ Texas_S-PDSO-N10_EP1.2x2.0mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+WDFN-8-1EP_2x2mm_P0.5mm_EP0.8x1.2mm:
+  device_type: 'WDFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://ww1.microchip.com/downloads/en/DeviceDoc/20005474E.pdf#page=25'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    nominal: 2.0
+    tolerance: 0
+  body_size_y:
+    nominal: 2.0
+    tolerance: 0
+  lead_width_min: 0.2
+  lead_width_max: 0.3
+  lead_len_min: 0.25
+  lead_len_max: 0.35
```

Comment:
This style is deprecated.
Please use:
```
lead_width: 0.2 ..0.25 ..0.3
lead_len: 0.25 ..0.3 .. 0.35
```

Or

```
lead_width: 0.25 +/- 0.05
lead_len: 0.3 +/- 0.05
```

see https://github.com/pointhi/kicad-footprint-generator/blob/master/scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/Readme.md